src/depots/loader/DepotLoader.ts

```diff
@@ -0,0 +1,46 @@
+import { Depot, ProtonRequired } from 'src/depots/loader/Depot';
+import R2Error from 'src/model/errors/R2Error';
+import Game from 'src/model/game/Game';
+
+export default class DepotLoader {
+
+    public static DEPOT_DEFAULT_KEY = "default";
+
+    private static LOADED_DEPOTS: Map<string, Depot[]> = new Map();
+
+    private static load(): Map<string, Depot[]> {
+        const master = require('../master.json');
+        console.log("Master:", master);
+        const depotMap = new Map<string, Depot[]>();
+        for (let depotsKey in master.depots) {
+            // @ts-ignore
```

Comment:
Is this really required? You could probably log it with ``console.log(`Depot key: ${depotsKey}`);`` without ts-ignore. Alternatively (and I'd argue the better option) you could explicitly type-hint the `master` variable on line 12